DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: rbabylon
 Type: Package
 Title: r package for babylon
-Version: 0.2.0.8100
+Version: 0.2.0.7101
```

Comment:
since this is a fix associated with the 0.3.0 release, this would increment the build number, not downgrade the version --> 8100 --> 8101.